share/tools/web_config/js/colorutils.js

```diff
@@ -393,19 +393,32 @@ function master_color_for_color(color_str) {
 
 /* Given a color name, like 'normal' or 'red' or 'FF00F0', return an RGB color string (or empty string) */
 function interpret_color(str) {
-	str = str.toLowerCase()
-	if (str == 'black') return '000000'
-	if (str == 'red') return 'FF0000'
-	if (str == 'green') return '00FF00'
-	if (str == 'brown') return '725000'
-	if (str == 'yellow') return 'FFFF00'
-	if (str == 'blue') return '0000FF'
-	if (str == 'magenta') return 'FF00FF'
-	if (str == 'purple') return 'FF00FF'
-	if (str == 'cyan') return '00FFFF'
-	if (str == 'white') return 'FFFFFF'
-	if (str == 'normal') return ''
-	return str
+    str = str.toLowerCase();
+    if (str == 'black') return '000000';
+    if (str == 'red') return '800000';
```

Comment:
Wouldn't it be nice here to be consistent with other programs, particularly terminals?

From [wikipedia](https://en.wikipedia.org/wiki/ANSI_escape_code#Colors), it seems X will use F00 for red, but 008000 for green and 00F for blue, like we currently do. Xterm is a bit weirder.